### DISCUSSION TITLE
Compatible ?

### DISCUSSION THREAD

**Discussion by jbesnard** [Yogui79/IntexPureSpa#5] [Q&A]
Hi, Is your system compatible with a C28453 ?

**Comment by LEECHER1** [comment]
Sorry, this model is not supported. Is not a 2020 version.

**Reply by jbesnard** [reply]
too bad for me, thanks for your quick answer. Good continuation in your great project.

**Comment by pulss** [comment]
HI,

is the model 28452NP compatible with the app?

Thanks Pavel
